actions/init/action.yml

```diff
@@ -0,0 +1,57 @@
+name: 'Wit Init'
+description: 'Initialize a wit workspace including the github project which invokes this action'
+inputs:
+  workspace:
+    description: |
+      The location where wit will initialize the workspace. All repositories,
+      including the invoking repository, will be subdirections of workspace.
+    required: false
+    default: '.'
+  environment:
+    description: |
+      The environment package to use when building this repository. This project
+      will be automatically added to the wit workspace
+    required: false
+    default: 'git@github.com:sifive/environment-example-sifive.git'
```

Comment:
1. Should this be called `environment` when it can work for any packages? I grant `environment` may better describe the pattern, but in theory you could have any number of packages here.

2. I don't think there should be a default

Reply:
Sure, I can remove the default.

If I could figure out how to accept a github action multiple times; ie:
```yml
- uses: sifive/wit/actions/init@v0.13.2
       with:
         extra: pkg1
         extra: pkg2
```

that would obviously be better. However, it does not appear that is possible yet. When/if they add that (seems github is actively improving this stuff) we should add another more general purpose option.

For now, for the use-cases we are actually catering towards, I think environment is a good choice.

Reply:
So the above syntax (repeated 'extra') is illegal and causes the yml file to be ignored. I've changed it to a space-separated additional_packages argument. It appears to work.